jukejoint_streamer/src/player/Player.ts

```diff
@@ -2,6 +2,7 @@ const lame = require('lame');
 const Speaker = require('speaker');
 const loudness = require('loudness');
 import * as ffmpeg from 'fluent-ffmpeg';
+import * as uuid from 'uuid';
```

Comment:
What's the use of using the `uuid` package?

Reply:
line 121: `song.id = uuid.v4();`

Reply:
What I meant was why are we using this package when it seems like the only thing it's doing is making sure it's a unique song, which we can do so via the property `url` already. Or am I missing something?

Reply:
it is necessary, because I think we should be able to add the same song to the playlist twice. in other services, such as google play and spotify, you are able to do that - hence I think we should support that too

Reply:
I agree, but I thought that some people might want to add a duplicate song to a playlist. Not a real use case though. I'll edit it out